File: code-python/support/base64_test.py
import base64

# 原始字符串
data = "Hello, Base64!匡正"

# 编码
encoded = base64.b64encode(data.encode()).decode()
print("Encoded:", encoded)

# 解码
decoded = base64.b64decode(encoded).decode()
print("Decoded:", decoded)

File: code-golang/support/base64_test.go
package support

import ("testing";"fmt";"encoding/base64")

func TestBase64(t *testing.T) {
	t.Log("testBase64")

	data := "Hello, Base64!匡正"

	// 编码
	encoded := base64.StdEncoding.EncodeToString([]byte(data))
	fmt.Println("Encoded:", encoded)

	// 解码
	decodedBytes, err := base64.StdEncoding.DecodeString(encoded)
	if err != nil {
		fmt.Println("Decode error:", err)
		return
	}
	fmt.Println("Decoded:", string(decodedBytes))
}